### PR TITLE
VZ-6337: Support flag --include-namespaces in the bug-report

### PR DESCRIPTION
**Pull request by bhatpmk**
This PR collects all the resources - Verrazzano resource, logs from pods and other resources from various namespaces in parallel. A change is also made to support multiple --include-namespaces flags for the bug-report command

### CODE REVIEW

File: tools/vz/pkg/bugreport/genreport.go [hunk 1]
```diff
@@ -98,58 +86,73 @@ func GenerateBugReport(kubeClient kubernetes.Interface, dynamicClient dynamic.In
...
+	// Report errors (if any), in collecting resources from various namespaces
+	for err := range ecr {
+		return fmt.Errorf("an error occurred while capturing the resource, error: %s", err.ErrorMessage)
+	}
 	return nil
 }
 
-// captureLogsInParallel collects the log from pods in parallel
-func captureLogsInParallel(wg *sync.WaitGroup, ec chan ErrorsChannel, kubeClient kubernetes.Interface, pods []corev1.Pod, namespace, bugReportDir string, vzHelper pkghelpers.VZHelper) {
+// captureVZResource collects the Verrazzano resource as a JSON
```
Comment (agv426) [inline tools/vz/pkg/bugreport/genreport.go:145]: consistency nit: indicate it is done in parallel

File: tools/vz/pkg/constants/constants.go [hunk 2]
```diff
@@ -104,9 +104,8 @@ const (
 	BugReportFileFlagUsage = "The report file to be created by bug-report command, as a .tar.gz file [Required]"
 
 	BugReportIncludeNSFlagName  = "include-namespaces"
-	BugReportIncludeNSFlagValue = ""
 	BugReportIncludeNSFlagShort = "i"
-	BugReportIncludeNSFlagUsage = "One or more namespaces to collect information from the cluster"
+	BugReportIncludeNSFlagUsage = "A comma separated list of namespaces to collect information from the cluster. This flag can be specified multiple times like --include-namespaces ns1 --include-namespaces ns..."
```
Comment (agv426) [inline tools/vz/pkg/constants/constants.go:108]: Thinking we need to clarify that these are additional namespaces that will be included, it is not limiting the namespaces captured to the list specified here.

Maybe "A comma separated list of **additional** namespaces..." is enough to clarify that

Comment (bhatpmk) [reply tools/vz/pkg/constants/constants.go]: Done

File: tools/vz/pkg/bugreport/genreport.go (hunk 1 above)
Comment (bhatpmk) [reply tools/vz/pkg/bugreport/genreport.go]: Done

**Review by agv426 — APPROVED** [review]

**Pull request merged**